web/packages/teleterm/src/ui/HeadlessAuthn/HeadlessPrompt/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2021 Gravitational, Inc.
```

Comment:
Year.

Reply:
FWIW, `make fix-license` works for code in web as well.